content/en/docs/setup/additional-setup/sidecar-injection/index.md

```diff
@@ -153,202 +153,94 @@ sleep-776b7bcdcd-bhn9m   2/2       Terminating   0          2m
 sleep-776b7bcdcd-gmvnr   1/1       Running       0          2s
 {{< /text >}}
 
-#### Understanding what happened
+#### Controlling the injection policy
 
-When Kubernetes invokes the webhook, the [`admissionregistration.k8s.io/v1beta1#MutatingWebhookConfiguration`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#mutatingwebhookconfiguration-v1beta1-admissionregistration-k8s-io)
-configuration is applied. The default configuration injects the sidecar into
-pods in any namespace with the `istio-injection=enabled label`. The
-`istio-sidecar-injector` configuration map specifies the configuration for the
-injected sidecar. To change how namespaces are selected for injection, you can
-edit the `MutatingWebhookConfiguration` with the following command:
+In the above examples, you enabled and disabled injection at the namespace level. Injection can also be controlled
+on a per-pod basis, by configuring the `sidecar.istio.io/inject` label on a pod:
 
-{{< text bash >}}
-$ kubectl edit mutatingwebhookconfiguration istio-sidecar-injector
-{{< /text >}}
+| Resource | Label | Enabled value | Disabled value |
+| -------- | ----- | ------------- | -------------- |
+| Namespace | `istio-injection` | `enabled` | `disabled` |
+| Pod | `sidecar.istio.io/inject` | `"true"` | `"false"` |
 
-{{< warning >}}
-You should restart the sidecar injector pod(s) after modifying
-the `MutatingWebhookConfiguration`.
-{{< /warning >}}
+The injector is configured with the following logic:
 
-For example, you can modify the `MutatingWebhookConfiguration` to always inject
-the sidecar into every namespace, unless a label is set. Editing this
-configuration is an advanced operation. Refer to the Kubernetes documentation
-for the [`MutatingWebhookConfiguration` API](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#mutatingwebhookconfiguration-v1beta1-admissionregistration-k8s-io)
-for more information.
+1. If either label is disabled, the pod is not injected
+1. If either label is enabled, the pod is injected
```

Comment:
And no label is marked as disabled

Reply:
This was supposed to be an ordered list, with checks performed in order. I was wondering if it would not come off that way, apparently it does not :slightly_smiling_face:

Reply:
Took me a while to get it :)